en/ros/ros2_offboard_control.md

```diff
@@ -0,0 +1,193 @@
+# ROS 2 Offboard Control Example
```

Comment:
Should it be ROS2 (rather than ROS 2)?

Reply:
ROS 2 (https://index.ros.org/doc/ros2/).